### PR TITLE
add webpack 4 & relay support for storybook

### LINKED ISSUE #2260: Fix Storybook 📖
## Issue - Bug

Running `yarn storybook` results in errors.

### Acceptance Criteria
- Updates `storybook` version in `package.json`
- Updates `.storybook/` configs as needed
- Adds `.storybook/.babelrc` config if needed

- **Estimated effort:** 10.5 points ([see CONTRIBUTING.md](https://github.com/ParabolInc/action/blob/master/CONTRIBUTING.md#points-and-sizes))

### PR DESCRIPTION
**Pull request by mattkrick**
storybook was broken in 3 fun ways:
it didn't support webpack 4
it didn't support our babel config
it didn't support relay

now it does! fixes #2260

TEST
- [ ] run storybook (note some of the stories are stale & the props have since changed!)

### CODE REVIEW

File: stories/components/RelayStub.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+/* @flow */
```
Comment (jordanh) [inline stories/components/RelayStub.js]: Nice! Love it!!

**Comment by jordanh** [comment]
# TEST
- [ ] run storybook (note some of the stories are stale & the props have since changed!)

## Notes

**Comment by jordanh** [comment]
LGTM!

**Comment by jordanh** [comment]
Note @ackernaut: might want to clean up what's not working (either by removing, or fixing if simple. No rush though.)

**Comment by mattkrick** [comment]
like 30 mins of just trial & error. relay has these funny functions that are basically intesnse duck-type checkers so I just used those as guidance on what to add

**Pull request merged**